linode/manager
# feat: [M3-6818] - Add AGLB Details - Summary Page

## Description 📝
- Adds AGLB details summary page

## Major Changes 🔄
- Updates Tabs
- Adds Details summary tab basics

## Preview 📷
![Screenshot 2023-08-16 at 12 48 56 PM](https://github.com/linode/manager/assets/115251059/1898d096-8b9f-4df9-bb57-c2a7d34e6582)

## How to test 🧪
- Navigate to `http://localhost:3000/loadbalancers/1`
- Verify the page is somewhat similar to the mockup
  - It's not a perfect re-creation of the mockup but this PR does setup the tabs and get a basic summary setup. We can fine tune this as a sub-task
- Check the responsiveness
